invenio_app_ils/ext.py

```diff
@@ -70,6 +79,8 @@ def init_app(self, app):
                 template_folder="templates"
             )
         )
+        # disable warnings being logged to Sentry
+        logging.getLogger("py.warnings").propagate = False
```

Comment:
I think this should go to Invenio-Logging.

Reply:
Yes, that would make more sense. I will create an issue for this in Invenio-Logging.